mono/mini/genmdesc.c

```diff
@@ -220,7 +220,7 @@ init_table (void) {
 
 static void
 output_char (FILE *f, char c) {
-	if (isalnum (c))
+	if (isalnum ((unsigned char)c)) // Cast to unsigned char to work around a bug in MSVCRT in debug builds
```

Comment:
I'm looking at the data the genmdesc tool consumes and it does rather look like getting data other than low-order ASCII in that `spec` field indicates a serious bug. There's little safety on the parsing so it might just be the file is incorrect.